integration/systemd/ecs.config

```diff
@@ -0,0 +1,2 @@
+ECS_LOGLEVEL=info
+ECS_CLUSTER=default
```

Comment:
I'm not sure that it makes sense for this file to be here. There's nothing systemd-specific about it, so at the very least it probably shouldn't be in the systemd subdirectory.

Reply:
Good call-out, fixing now.